demo/admin/src/dam/getRandomUnsplashImage.ts

```diff
@@ -34,7 +42,11 @@ export async function getRandomUnsplashImage(): Promise<UnsplashImage> {
         }
 
         const fileName = extractFileNameFromUrl(image.origin);
-        const acceptedFile = new File([image.blob], fileName, { type: mimeType });
+        const acceptedFile = new ExtendedFile([image.blob], fileName, { type: mimeType });
+        acceptedFile.importSource = {
+            importSourceId: image.origin,
+            importSourceType: "External source",
```

Comment:
Source type should be unsplash, as before.

Reply:
0f475a5abb42735bb4897004df52e8a6a2757651